backend/ideas/models/translation.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+
+from ideas.models import Idea
+from climateconnect_api.models.language import Language
+
+
+class IdeaTranslation(models.Model):
+    idea = models.ForeignKey(
+        Idea,
+        help_text="Points to idea table",
+        verbose_name="Idea",
+        related_name="translate_idea",
+        on_delete=models.CASCADE
+    )
+
+    language = models.ForeignKey(
+        Language,
+        help_text="Points to the language idea is translated for",
+        verbose_name="Language",
+        related_name="idea_language",
+        on_delete=models.CASCADE
+    )
+
+    name = models.CharField(
```

Comment:
I would keep the convention of having `[fieldName]_translation` here, e.g. `name_translation` in this case

Reply:
Done